### PR TITLE
System notification navbar menu drop down [SCI 2955]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: https://biosistemika.atlassian.net/browse/SCI-2955

### What was done
Added dropdown menu to navbar with system notifications and specs

### CODE REVIEW

File: app/models/user_system_notification.rb [hunk 1]
```diff
@@ -16,6 +18,16 @@ def self.mark_as_read(notification_id)
     end
   end
 
+  def self.unseen_count
+    unseen.count
+  end
+
+  def self.modal(notification_id)
```
Comment (urbanrotnik) [inline app/models/user_system_notification.rb]: Didn't you move this to scope?

Comment (aignatov-bio) [reply app/models/user_system_notification.rb]: I think maybe remove at all, cause we have unseen scope. And if we need find unseen count, we can simply use unseen.count.

File: spec/models/system_notification_spec.rb [hunk 2]
```diff
@@ -69,4 +70,44 @@
...
+             system_notification: notifcation_one
+      create :user_system_notification,
+             user: user,
+             system_notification: notifcation_two
+    end
+
+    it 'get last notifications without search' do
+      result = SystemNotification.last_notifications(user)
+      expect(result.length).to eq 2
+      expect(result.first).to respond_to(
```
Comment (urbanrotnik) [inline spec/models/system_notification_spec.rb]: This test doesn't make sense here I think

Comment (aignatov-bio) [reply spec/models/system_notification_spec.rb]: It's more for test right result format. Cause if response structure will be changed, system notifications view may failed.

File: app/models/user_system_notification.rb (hunk 1 above)
Comment (radiokills) [reply app/models/user_system_notification.rb]: @aignatov-bio Yes, that's a great idea.

**Pull request merged**